l10n_es_ticketbai_api/models/ticketbai_response.py

```diff
@@ -0,0 +1,224 @@
+                                    "eu_ES": xml_dict["Salida"]["Azalpena"],
+                                },
+                            },
+                        )
+                    ]
+                messages = xml_dict.get("Salida").get("ResultadosValidacion", False)
+                if messages:
+                    if isinstance(messages, dict):
+                        messages = [messages]
+                    for msg in messages:
+                        tbai_response_message_ids.append(
+                            (
+                                0,
+                                0,
+                                {
+                                    "code": msg["Codigo"],
+                                    "description": {
+                                        "es_ES": msg["Descripcion"],
+                                        "eu_ES": msg["Azalpena"],
+                                    },
+                                },
+                            )
+                        )
+            elif state == TicketBaiResponseState.REJECTED.value:
+                messages = xml_dict["Salida"]["ResultadosValidacion"]
+                if isinstance(messages, dict):
+                    messages = [messages]
+                for msg in messages:
+                    tbai_response_message_ids.append(
+                        (
+                            0,
+                            0,
+                            {
+                                "code": msg["Codigo"],
+                                "description": {
+                                    "es_ES": msg["Descripcion"],
+                                    "eu_ES": msg["Azalpena"],
+                                },
+                            },
+                        )
+                    )
+            else:
+                tbai_response_message_ids.append(
+                    (
+                        0,
+                        0,
+                        {
+                            "code": state,
+                            "description": _("Unknown TicketBAI response code."),
+                        },
+                    )
+                )
+            values.update(tbai_response_message_ids=tbai_response_message_ids)
+        return values
+
+
+class TicketBaiResponseMessage(models.Model):
+    _name = "tbai.response.message"
+    _description = "TicketBAI Tax Agency Invoice response messages"
+
+    tbai_response_id = fields.Many2one(
+        comodel_name="tbai.response", required=True, ondelete="cascade"
+    )
+    code = fields.Char(required=True)
+    description = fields.Char(required=True, translate=True)
+
+    @api.depends("code", "description")
+    def name_get(self):
+        result = []
+        for msg in self:
+            name = "{} - {}".format(msg.code, msg.description)
+            result.append((msg.id, name))
+        return result
+
+    @api.model
+    def create(self, vals):
```

Comment:
El método create tendría que pasarse a  @api.model_create_multi, esto arroja warning en el log.

2023-05-30 20:32:14,604 255 WARNING odoo odoo.api.create: The model odoo.addons.l10n_es_ticketbai_api.models.ticketbai_response is not overriding the create method in batch